modApiServer/src/org/aion/api/server/types/TxRecptLg.java

```diff
@@ -56,7 +55,7 @@
         this.blockHash = b == null ? null : TypeConverter.toJsonHex(b.getHash());
         this.transactionIndex = (b == null || txIndex == null) ? null : TypeConverter.toJsonHex(txIndex);
         this.transactionHash = TypeConverter.toJsonHex(tx.getHash());
-        this.address = (tx == null || tx.getTo() == null) ? "" : TypeConverter.toJsonHex(tx.getTo().toBytes());
+        this.address = TypeConverter.toJsonHex(logInfo.getAddress().toString());
```

Comment:
did you change this intentionally?  I thought Jay already made this change (not sure why it shows up in the PR)

Reply:
yeah, i'm not sure what's going on here either? afaic tell, this file is identical in both master-pre-merge and test-TxReLog